platform/commonUI/edit/src/capabilities/EditorCapability.js

```diff
@@ -81,29 +81,28 @@ define(
         };
 
         /**
-         * Save any changes from this editing session. This will flush all
-         * pending persists and end the current transaction
+         * Save any unsaved changes from this editing session. This will
+         * end the current transaction and continue with a new one.
          * @returns {*}
          */
         EditorCapability.prototype.save = function () {
-            var domainObject = this.domainObject;
-            return this.transactionService.commit().then(function () {
-                domainObject.getCapability('status').set('editing', false);
+            var transactionService = this.transactionService;
+            return transactionService.commit().then(function () {
+                transactionService.startTransaction();
             });
         };
 
         EditorCapability.prototype.invoke = EditorCapability.prototype.edit;
 
         /**
-         * Cancel the current editing session. This will discard any pending
+         * Finish the current editing session. This will discard any pending
          * persist operations
          * @returns {*}
          */
-        EditorCapability.prototype.cancel = function () {
+        EditorCapability.prototype.finish = function () {
             var domainObject = this.domainObject;
             return this.transactionService.cancel().then(function () {
                 domainObject.getCapability("status").set("editing", false);
-                return domainObject;
```

Comment:
Gotcha, the change has been reverted. I removed it initially because it didn't appear to be used anymore at least after a few searches.